### DISCUSSION TITLE
Filter releases by type

### DISCUSSION THREAD

**Discussion by mikesir87** [community/community#5286] [Other Feature Feedback, Questions, & Ideas] [answered]
Quite often when I am performing updates to various components, I will look at a repo's releases to look at release notes. For many projects, they have numerous beta/RC releases, which makes scanning releases much more difficult. Fortunately, most repos are good about indicating they are "Pre-releases". 

To help me scan the releases for info I need to read, I would like to have the ability to filter out "Pre-release" releases.

**Comment by MylesBorins** [comment, marked as answer]
@mikesir87 this is now possible via search!

Check out [the docs](https://docs.github.com/en/repositories/releasing-projects-on-github/searching-a-repositorys-releases)

**Reply by lrworth** [reply]
Any plans to make search available in the API / `gh` tool?

**Reply by pavinduLakshan** [reply]
This doesn't seem to work now. Neither of these queries return any results.

https://github.com/changesets/changesets/releases?q=tag%3Av4&expanded=true
https://github.com/changesets/changesets/releases?q=tag%3A%40changesets%2Fget-release-plan&expanded=true

**Comment by mikesir87** [comment]
Fantastic! Thank you @MylesBorins! 🎉
